sources.cfg

```diff
@@ -5,3 +5,7 @@ extends =
 extensions += mr.developer
 
 auto-checkout =
+    ftw.bumblebee
+
+[sources]
+ftw.bumblebee = git git@github.com:4teamwork/ftw.bumblebee.git
```

Comment:
This source should be placed in the [kgs sources.cfg](https://github.com/4teamwork/kgs/tree/master/sources) and not repeated in the packages. Then this line can be dropped.